dist/obs/deploy_to_obs.sh.in

```diff
@@ -4,9 +4,24 @@ PACKAGE_NAME=strusbindings
 PACKAGE_VERSION=@STRUS_VERSION@
 OSC_HOME=$HOME/home:PatrickFrey/$PACKAGE_NAME
 
-rm -f ${PACKAGE_NAME}-${PACKAGE_VERSION}.tar.gz
-cmake .
-make dist-gz
+TMPDIR=/tmp
+
+if test "x$1" = "xtest"; then
```

Comment:
git-archive packs only things which are committed.
a test mode here which packs all of the current workspace is handy, though dangerous, as if things
get to OBS, we make a mess. :-)